lib/jsdoc/util/logger.js

```diff
@@ -130,34 +141,26 @@ function addPrefix(args, prefix) {
     return updatedArgs || args;
 }
 
-// TODO: document events
-function wrapLogFunction(name, func) {
-    var eventName = 'logger:' + name;
-    var upperCaseName = name.toUpperCase();
-    var level = LEVELS[upperCaseName];
-    var prefix = PREFIXES[upperCaseName];
-
-    return function() {
-        var loggerArgs;
 
-        var args = Array.prototype.slice.call(arguments, 0);
-
-        if (logLevel >= level) {
-            loggerArgs = addPrefix(args, prefix);
-            func.apply(null, loggerArgs);
-        }
+var Logger = function () {
+    this.handlers = {};
```

Comment:
Sorry for the confusion. Here's what I'm suggesting:

+ By default, the logger is just an event emitter with no listeners. This means we would not call `setDefaultHandlers` within this module.
+ If you want to log messages to the console, you simply call `setDefaultHandlers`. JSDoc's CLI would need to call this method. (In the future, when JSDoc has a programmatic interface, the code path for the programmatic interface would _not_ call this method.)
+ If you want to do something else with log messages, you attach a listener, just as you would do with any emitter. The `Logger` instance does not do anything else to keep track of event handlers--they're just normal listeners.

Does that make more sense?